### PR TITLE
Feature/1746 add cart scraper functionality to the js library for electron

### PR DESCRIPTION
**Pull request by kernel24**
## Overview

This PR mainly adds new functionality as cart scraper to the electron app. Also, changes the installation to download the package via npm

- add cart scraper functionality to the electron
- add build option for Windows
- updates readme
- updates license

## Version
v0.1.4

### CODE REVIEW

**Review by hades-6 — APPROVED** [review]
looks amazing 💯

File: electronjs/readme.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
 ### Installation
 
 ```bash
-$> npm install fillr-extension --registry https://api.bintray.com/npm/fillr/npm
-$> npm install
+$> npm i @fillr_letspop/desktop-autofill
```
Comment (hades-6) [inline electronjs/readme.md:14]: maybe explain a little how where to get the username and password? or that they should contact us?

Comment (kernel24) [reply electronjs/readme.md:14]: @hades-6  @fillr_letspop/desktop-autofill is public, will add permission note for @fillr_letspop/cart-scraper which is private package

File: electronjs/fillr.js [hunk 2]
```diff
@@ -15,9 +15,17 @@ const interval = setInterval(() => {
...
     })
-    
     const fillr = new FillrController.default(devKey, secretKey, profileDataHandler);
+
+    const FillrScraper = require('@fillr_letspop/cart-scraper')
+    window.FillrCartInformationExtractionInterface.setDevKey(devKey);
+    const onCartDetected = function(ev) {
+
+      const cartInfo = ev.detail;
+      alert(JSON.stringify(cartInfo));
```
Comment (mhussain) [inline electronjs/fillr.js]: Please add the error handling for JSON

**Review by mhussain — CHANGES_REQUESTED** [review]

**Review by hades-6 — APPROVED** [review]
👏

File: electronjs/fillr.js (hunk 2 above)
Comment (kernel24) [reply electronjs/fillr.js]: @mhussain This is just for the example showing the log.

**Review by mhussain — APPROVED** [review]

**Review by mhussain — APPROVED** [review]

**Review by hades-6 — APPROVED** [review]

**Pull request merged**